skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevmetadataService.kt

```diff
@@ -27,15 +27,18 @@ class BrevmetadataService(config: Config) {
         }
     }
 
-    suspend fun getRedigerbareBrev(sakstype: String): List<LetterMetadata> {
+    suspend fun getRedigerbareBrev(sakstype: String, includeVedtak: Boolean): List<LetterMetadata> {
         val httpResponse = httpClient.get("/api/brevdata/brevdataForSaktype/$sakstype?includeXsd=false") {
             contentType(ContentType.Application.Json)
         }
 
         if (httpResponse.status.isSuccess()) {
             return httpResponse.body<List<BrevdataDto>>()
                 .filter { it.redigerbart }
-                .filter {it.brevkategori != BrevdataDto.BrevkategoriCode.VEDTAK}
+                .filter { when (includeVedtak) {
```

Comment:
Finnes sikkert noe mer Kotlinesque måte å gjøre dette?

Reply:
Vanlig logikk dette. .filter{ it.brevkategori != BrevdataDto.BrevkategoriCode.VEDTAK || includeVedtak }